src/ontology/edits/oeo-physical.omn

```diff
@@ -9262,6 +9262,121 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
         OEO_00140104
     
     
+Class: OEO_00310023
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A heat recovery steam generator is a heat exchanger that recovers heat from a hot steam.",
```

Comment:
> A heat recovery steam generator is a heat exchanger that recovers heat from a hot steam.

I would expect a "heat recovery steam generator" to generate steam from heat that has been recovered (as opposed to heat generated specifically for steam generation).  That is also what happens in a CCGT – hot off-gas from the gas turbine is used to generate steam.

Reply:
Yes, I think I had a wrong understanding here. I propose a new definition:
`A heat recovery steam generator is a heat exchanger that uses recovered heat to generate steam.`
I this correct?

Reply:
Guess so.
Do we generally track the water?  Does the ontology condensers and feed pumps to track the entire water cycle?  Unless the `uses some water` is needed right now for some application, I would probably skip it.

Reply:
We don't necessarily need the water as part of the definition. So what about: _A heat recovery steam generator is a heat exchanger that uses recovered heat to generate steam._ However, I think we should still axiomatise the water, even if is not part of the definiton:
* `uses some water`
* `'participates in' some ('heat transfer' and 'has physical input' some 'liquid water' and 'has physical output' some 'steam')`

Reply:
> We don't necessarily need the water as part of the definition. So what about: _A heat recovery steam generator is a heat exchanger that uses recovered heat to generate steam._ However, I think we should still axiomatise the water, even if is not part of the definiton:

Sounds fine to me 👍